.github/workflows/interface-unit-tests.yml

```diff
@@ -12,6 +12,12 @@ on:
       codecov_token:
         description: The codecov token to use when uploading coverage files
         required: true
+      pennylane_oss_server_endpoint_url:
+        description: The endpoint URL for the PennyLane OSS Server
```

Comment:
Just wondering, assuming "OSS" means "open-source software" should we name this by what it does/what it is for, rather than who uses it?
I don't think I could clearly identify what "PennyLane OSS Server" is coming in here without some background understanding.